client/client_test.go

```diff
@@ -199,15 +200,16 @@ func TestCanPerformTLS(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
+		item := tt
```

Comment:
Same comment as above - for consistency, I'd rename `tests` to scenarios`, get rid of `item := tt` and rename `tt` to `scenario`